src/ue.cpp

```diff
@@ -150,6 +150,12 @@ static nr::ue::UeConfig *ReadConfigYaml()
 
     if (yaml::HasField(config, "supi"))
         result->supi = Supi::Parse(yaml::GetString(config, "supi"));
+    if (yaml::HasField(config, "protectionScheme"))
+        result->protectionScheme = yaml::GetInt32(config, "protectionScheme", 0, 255);
+    if (yaml::HasField(config, "homeNetworkPublicKeyId"))
+        result->homeNetworkPublicKeyId = yaml::GetInt32(config, "homeNetworkPublicKeyId", 0, 255);
+if (yaml::HasField(config, "homeNetworkPublicKey"))        
```

Comment:
Please fix the indentation in line 157.

Reply:
It will be fixed. Sorry for this basic mistake :o(